introduction.rst

```diff
@@ -1,3 +1,225 @@
 ============
 Introduction
 ============
+
+Rationale
+---------
+
+The Radicalbit Fast Data Distribution (RBFDD) is a collection of Open Source (`ASL 2.0-licenced <https://www.apache.org/licenses/LICENSE-2.0.html>`_) software products connecting, leveraging and exploiting data, network, intelligence and computing power.
+
+RBFDD includes modern, present and future-proof technologies, fully integrated, forming an end-to-end data pipeline computing environment in which a pool of software, leveraging computers collaborating over the network, takes part to solve complex problems in order to achieve a common goal.
+
+RBFDD is a seamlessly integrated computing layer enabling organizations to develop and run business logic such as:
+
+* statistical analysis
+* predictive and prescriptive analysis
+* log analytics
+* anomaly detection
+* smart metering
+* fraud detection
+* anti-money laundering
+* graph processing
+* industrial internet (IoT)
+* any analytics task requiring speed at scale
+
+in a fast, event-driven, scalable, resilient, secure, highly available and integrated environment.
+
+RBFDD supports both batch, long-running jobs, and streaming, pure real-time processing, leveraging concepts such as parallel, distributed, and in-memory computing in order to process massive amount of data and workload at speed.
+
+RBFDD acts as a computing layer, service or ingestion at your will, providing organizations the ability to develop, integrate and deploy services requiring complex calculations, data intensive computation or real-time processing.
+
+RBFDD allows implementing data-driven, service-oriented systems, by quickly get up and running with one of the most advanced piece of software technology currently available on the market.
+
+Components overview
+-------------------
+
+The following diagram displays the ecosystem of software components in which RBFDD lives; in blue are components part of the distribution, while in green are key components that, while not being provided along with the distribution, represent a key feature item for which integrations (across several dimensions) are provided, or that are otherwise regarded as a first-class citizen in the distribution:
+
+.. image:: /img/introduction/blueprint.svg
+   :height: 400px
+   :width: 640px
+   :alt: RBFDD Blueprint
+   :align: center
+
+.
```

Comment:
I couldn't find any other way to put some padding between the image and the rest of the chapter, any help with this would be appreciated.